src/Veldrid/Vk/VkGraphicsDevice.cs

```diff
@@ -468,6 +468,11 @@ private void CreateInstance(bool debug, VulkanDeviceOptions options)
             StackList<IntPtr, Size64Bytes> instanceExtensions = new StackList<IntPtr, Size64Bytes>();
             StackList<IntPtr, Size64Bytes> instanceLayers = new StackList<IntPtr, Size64Bytes>();
 
+            if (availableInstanceExtensions.Contains(CommonStrings.VK_KHR_portability_subset))

```

Comment:
Not sure if this or the other similar change below is necessary.  Will need to double-check.

Reply:
(whichever one I did first didn't work alone)

Reply:
This doesn't seem like it should be necessary, and I don't think this extension is finalized yet, so I'd rather not use it even optionally.